src/components/ShowPage.js

```diff
@@ -20,16 +20,40 @@ import Error from './Error';
 import ShowRespond from './ShowRespond';
 import ShowResults from './ShowResults';
 import ShareModal from './ShareModal';
+import Modal from 'react-modal';
+import './ReactModalOverride.scss';
+
```

Comment:
Need to run prettify again

Reply:
Prettify at least for me does not remove double empty lines if thats what youre referring to

Reply:
How are you running Prettier? `yarn prettify` should remove the double empty lines. Maybe you are running a different formatter?